moto/ec2/models.py

```diff
@@ -2995,6 +2995,29 @@ def create_vpc_endpoint(
 
         return vpc_end_point
 
+    def get_vpc_end_point_services(self):
+        vpc_end_point_services = self.vpc_end_points.values()
+
+        services = []
+        for value in vpc_end_point_services:
+            services.append(value.service_name)
+            print(".".join(reversed(value.service_name.split("."))))
+
+        availability_zones = [
```

Comment:
These don't have to be hardcoded:
https://github.com/spulec/moto/blob/2504a398f9ac9659986abaeb0ba19d6550eeb309/moto/ec2/models.py#L1811